src/spyglass/position_linearization/position_linearization_merge.py

```diff
@@ -0,0 +1,40 @@
+import datajoint as dj
+
+from spyglass.common.common_nwbfile import AnalysisNwbfile
+from spyglass.position_linearization.v1.linearization import (
+    LinearizedPositionV1,
+)  # noqa F401
+from spyglass.utils.dj_helper_fn import fetch_nwb
+
+schema = dj.schema("position_linearization_merge")
+
+
+@schema
+class LinearizedPositionOutput(dj.Manual):
+    definition = """
+    linearized_position_id: uuid
+    ---
+    source: varchar(40)
+    version: int
+
+    """
+
+    class LinearizedPositionV1(dj.Part):
+        definition = """
+        -> master
```

Comment:
I'm not sure if this is still the case (@CBroz1, @rly), but there have been bugs in the past due to using `master` instead of `TableName` here. I'm specifically thinking about datajoint/datajoint-python/pull/991, but probably isn't a huge deal since Ryan fixed that one.